craft_cli/helptexts.py

```diff
@@ -325,6 +326,9 @@ def _build_plain_command_help(
         )
 
         overview = textwrap.indent(command.overview, "    ")
+        # Remove reST-style double backticks
+        # Match _only_ double backticks, never triples
+        overview = re.sub(r"(?<!`)``(?!`)", "", overview)
```

Comment:
Normally I would suggest that we `re.compile` this into a global constant, but I think in this case this method should only ever run once per process invocation so it's probably not worth it?

Reply:
I think it actually might make a separate call to this function for each field it's validating, so I'll move it to a constant. Good catch!